### ISSUE TITLE
Slider doesn't resize on window resize

### ISSUE THREAD

**Issue by dgsiegel** [maxshuty/accessible-web-components#4]
If you resize any example on the docs page, the slider will not adjust its width automatically:

![image](https://user-images.githubusercontent.com/4396895/155427600-68faf41a-6a67-4add-9ba9-c72843faddac.png)

I'd suggest a `resizeObserver` or binding to the window `resize` event.

**Comment by maxshuty** [comment]
Thanks for reporting @dgsiegel. A fix for this is currently in progress. I think this is a perfect use case for the `resizeObserver`.

**Comment by KonstantinosPetrakis** [comment]
First of all, I would like to underline the great work you've done with the project to bring something that in my opinion is missing from the standard HTML. 

I am currently working in a strict deadline project and I don't have the time to read the internals of the project and contribute despite how much I would want that. Although because I actually plan to use this in my current project I have come up with a dirty workaround solution that others might want to use as the issue is being resolved. I am sure if a relatively similar thing it's done internally it wield better performance. Although for those that can't wait this can help.

```js
document.addEventListener("DOMContentLoaded", () => {
    const resizeObserver = new ResizeObserver(_ => {
        for (let oldRange of document.querySelectorAll("range-selector")) {
            let minInputValue = oldRange.shadowRoot.querySelector("input[name='min']").getAttribute("data-value");
            let maxInputValue = oldRange.shadowRoot.querySelector("input[name='max']").getAttribute("data-value");
            let newRange = document.createElement("range-selector");
            for (let attr of oldRange.attributes) newRange.setAttribute(attr.name, attr.value);
            newRange.setAttribute("preset-min", minInputValue);
            newRange.setAttribute("preset-max", maxInputValue);
            oldRange.parentNode.replaceChild(newRange, oldRange);
        }
    });    
    resizeObserver.observe(document.body);
});
```

**Comment by maxshuty** [comment]
@KonstantinosPetrakis as I'm actively contributing to this project I'll try and get a release out this week to address this. This one has been hanging around for a while as I've got a few other projects I contribute to as well with higher priority items, but seeing now as there are multiple posts on this I'll tackle it. Thanks for the support!

**Comment by maxshuty** [comment]
@dgsiegel and @KonstantinosPetrakis this has been fixed in [**`v1.0.8`**](https://github.com/maxshuty/accessible-web-components/pull/9) using [**`ResizeObserver`**](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) which automatically resizes the range selector while maintaining it's existing state.
